koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -44,6 +45,8 @@
 from reporting.provider.ocp.models import OCPUsageReport
 from reporting.provider.ocp.models import OCPUsageReportPeriod
 
+# from reporting.provider.ocp.models import PRESTO_LINE_ITEM_TABLE_MAP
+
```

Comment:
I'm a little confused. Per our slack convo, I mentioned that the label processing was for unfiltered labels that the original processing was using the daily table to process. Since my new processing bypasses the daily stuff and just writes to summary, I thought you said to ignore this suggestion.
Or are you saying that I _should_ be processing daily tables as well? I just want to make sure I'm understanding the desired output from presto processing.

Reply:
No, we don't need to process the daily tables. AWS and Azure switched to using the daily summary table for tag summary. As tag enablement for AWS and Azure are new functionality we're starting with an add-on step to filter the daily summary table at the end to only have filtered tags. I just meant for now let's keep an eye on both methods as we get into testing this all out and see which how both work.